ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -36,7 +36,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.14
```

Comment:
it was announced in the mailing list that we should move to 1.15
```
[aos-devel] ACTION REQUIRED: Migrating OpenShift 4.6 to (a default) of golang 1.15
```

Reply:
done